PLAN.md

```diff
@@ -0,0 +1,57 @@
+# plan for opentelemetry-python-contrib
+This document captures the effort to move instrumentation and exporters out of the main repo and into the opentelemetry-python-contrib. Doing this will give us the ability to decouple changes to the core of OpenTelemetry (API/SDK) from changes to the rest of the eco-system. This pattern is utilized in other OpenTelemetry SIGs as well.
```

Comment:
```suggestion
This document captures the effort to move instrumentation and exporters out of the main repo and into `opentelemetry-python-contrib`. Doing this will give us the ability to decouple changes to the core of OpenTelemetry (API/SDK) from changes to the rest of the eco-system. This pattern is utilized in [other OpenTelemetry SIGs as well](https://github.com/open-telemetry?q=contrib&type=&language=).
```